addons/account/models/account_move.py

```diff
@@ -300,6 +300,8 @@ def _get_default_invoice_incoterm(self):
         help="Auto-complete from a past bill.")
     invoice_source_email = fields.Char(string='Source Email', tracking=True)
     invoice_partner_display_name = fields.Char(compute='_compute_invoice_partner_display_info', store=True)
+    is_inactive_currency = fields.Boolean(compute="_compute_is_inactive_currency",
```

Comment:
`display_inactive_currency_warning` ? Also move this field into the dedicated `# ==== Display purpose fields ====` section